AsyncDisplayKit/ASVideoPlayerNode.h

```diff
@@ -0,0 +1,147 @@
+//
+//  ASVideoPlayerNode.h
+//  AsyncDisplayKit
+//
+//  Created by Erekle on 5/6/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#if TARGET_OS_IOS
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+//#import <AsyncDisplayKit/ASThread.h>
+//#import <AsyncDisplayKit/ASVideoNode.h>
+//#import <AsyncDisplayKit/ASDisplayNode+Subclasses.h>
+
+@class AVAsset;
+@protocol ASVideoPlayerNodeDelegate;
+
+typedef enum {
```

Comment:
I'm a little concerned about the flexibility of adding additional nodes with this approach.  An example is that what if the client needs to add the sound control?  Or additional app-specific control components?